.eslintrc

```diff
@@ -8,13 +8,10 @@ module.exports = {
         "devDependencies": [
           "config/*.js",
           "**/*.test.jsx",
-          "**/*.test.js",
+          "**/*.test.js"
```

Comment:
Pretty sure this could just be one line: `**/*.test.jsx?`

Reply:
I tried that but it didn't like it - I'll try again